GG/GG/RichText/BlockControl.h

```diff
@@ -0,0 +1,29 @@
+#ifndef BLOCKCONTROL_H
+#define BLOCKCONTROL_H
+
+#include <GG/Control.h>
+
+namespace GG
+{
+
+/** \brief BlockControl is an abstract base class for controls that can determine their height
+ * when you set their width.
+ *
+ * BlockControls are used for embedding controls in text.
+ */
+class BlockControl : public GG::Control
+{
+public:
+    //! Create a block control.
+    BlockControl ( X x, Y y, X w, GG::Flags< GG::WndFlag > flags );
+
```

Comment:
please remove extra spaces and redundant GG:: namespace specification

`BlockControl(X x, Y y, X w, Flags<WndFlag> flags)`